src/servico/NotificacaoService.java

```diff
@@ -58,7 +60,11 @@ public String notificarAusencia(Long idProfessor, String dataInicio, String data
             inícioReal = new DateTime(inicio.getYear(),inicio.getMonthOfYear(),inicio.getDayOfMonth(),00,00);
 
             fim = new DateTime(sdf.parse(dataFim));
-            fimReal = new DateTime(fim.getYear(),fim.getMonthOfYear(),fim.getDayOfMonth(),23,59);            
+            
+            int hourOfDay = 23;
+            int minuteOfHour = 59;
+            
+            fimReal = new DateTime(fim.getYear(),fim.getMonthOfYear(),fim.getDayOfMonth(),hourOfDay,minuteOfHour);            

```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Line is longer than 80 characters (found 126).](https://www.codacy.com/app/calebepb/substituicao/file/2728205073/issues/source?bid=3308597&fileBranchId=3349120#l67)